src/pq.cpp

```diff
@@ -101,13 +115,26 @@ namespace diskann {
           __FILE__, __LINE__);
     }
 
+    if (use_old_filetype) 
```

Comment:
maybe cleaner to say something like if(old_format) chunk_offsets_location = 3 else chunk_offset_location = 2, and use it when load_bin?